src/Microsoft.ApplicationInsights.AspNetCore/DiagnosticListeners/Implementation/MvcDiagnosticsListener.cs

```diff
@@ -0,0 +1,86 @@
+namespace Microsoft.ApplicationInsights.AspNetCore.DiagnosticListeners
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.Extensions.DiagnosticAdapter;
+
+    public class MvcDiagnosticsListener : IApplicationInsightDiagnosticListener
+    {
+        internal const string BeforeActionNotificationName = "Microsoft.AspNetCore.Mvc.BeforeAction";
+
+        public string ListenerName { get; } = "Microsoft.AspNetCore";
+
+        [DiagnosticName(BeforeActionNotificationName)]
```

Comment:
I know it was copied. But to make it looks consistent with other diagnostic source callbacks - no need to use a constant here.